cmd/buildah/addcopy.go

```diff
@@ -14,7 +15,8 @@ var (
 		},
 		cli.StringFlag{
 			Name:  "root",
-			Usage: "root `directory` of the working container",
+			Usage: "root `directory` of working container, Default: EnvVar(BUILDAHROOT)",
+			Value: os.Getenv("BUILDAHROOT"),
```

Comment:
I think you can set `EnvVar: "BUILDAHROOT"` here and let the cli package handle the rest.